WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -518,6 +521,19 @@ private extension CardPresentPaymentsOnboardingUseCase {
     }
 }
 
+// MARK: - Analytics
+private extension CardPresentPaymentsOnboardingUseCase {
+    func trackPluginInstallFailed(_ error: Error) {
+        guard let countryCode = self.storeCountryCode else {
+            DDLogError("Error installing plugin: \(error)")
+            return
+        }
+        ServiceLocator.analytics.track(event: .InPersonPayments.cardPresentOnboardingCtaFailed(reason: "plugin_install_tapped",
```

Comment:
Yes, we use the `reason` as the the CTA step since we're building on top of the existing ones ( context: pdfdoF-3HA#comment-4930-p2 ) and then details of the error itself are logged into the error description, in this case since they come from the API:

```Dotcom Error: [plugins_api_failed] Plugin not found```

We'll use the reason as well for future CTA failure steps, like `plugin_activate_tapped`, `plugin_setup_tapped`, or 
 `stripe_account_setup_tapped`

Reply:
> I see these more like context, or trigger, than the reason why it failed. If we already tracked events with that key it's ok as it is, it's not worth to change it now :)

Agree!